proto/src/main/proto/Rss.proto

```diff
@@ -256,6 +256,9 @@ service CoordinatorServer {
 
   // Heartbeat between Shuffle Application and Coordinator Server
   rpc appHeartbeat(AppHeartBeatRequest) returns (AppHeartBeatResponse);
+
+  // Access to the remote shuffle service cluster
+  rpc accessCluster(AccessClusterRequest) returns (AccessClusterResponse);
```

Comment:
Do we need to return an "access token"? 
It can be used in all gRPC requests to grant the access.

Reply:
This is a one-time check to decide the shuffle strategy on the client-side no need to grant a token.